test/info.js

```diff
@@ -292,6 +292,19 @@ describe('Info operations', () => {
       const layer = FeatureServer.layerInfo(input, {})
       layer.fields[0].type.should.equal('esriFieldTypeOID')
     })
+
+    it('should override the default true value of "hasStaticData" when set in metadata', () => {
+      const input = {
+        metadata: {
+          hasStaticData: false,
+          geometryType: 'Polygon',
+          extent: [[11, 12], [13, 14]],
+          fields: [{ name: 'test', type: 'integer' }]
+        }
+      }
+      const layer = FeatureServer.layerInfo(input, {})
+      layer.hasStaticData.should.equal(false)
```

Comment:
has static data should default to false

Reply:
Changed default to `false`, made test adjustments, and noted in changelog in 1f2cb24636bbb1c1bec0e9fc207cf8ff6fe352fe.

Reply:
👍

Reply:
since this test is overriding, then it should be testing the `true` condition